ppr-ui/src/views/amendment/AmendRegistration.vue

```diff
@@ -651,6 +653,24 @@ export default class AmendRegistration extends Vue {
     this.debtorValid = val
   }
 
+  private validateSecuredParties (): void {
+    const sp = this.getAddSecuredPartiesAndDebtors.securedParties
+    let securedPartyCount = 0
+    if (this.registrationType === APIRegistrationTypes.SECURITY_AGREEMENT) {
```

Comment:
We can simplify further. There is no need to initialize the `sp` and `securedPartyCount` before we check for `registrationType`, and we can also set a default `securedPartyCount`.

```
if (this.registrationType === APIRegistrationTypes.SECURITY_AGREEMENT) {
   const sp = this.getAddSecuredPartiesAndDebtors.securedParties
   const securedPartyCount = sp.filter(removed => removed.action !== ActionTypes.REMOVED).length || 0
   this.setValidSecuredParties(securedPartyCount >= 1)
}
```

Reply:
Change made.